open-metadata-implementation/admin-services/admin-services-server/pom.xml

```diff
@@ -122,6 +122,13 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>data-engine-proxy-services-server</artifactId>
+            <version>${open-metadata.version}</version>
```

Comment:
version should be removed
Also why are common admin services now dependent on an OMAS server?

Reply:
the Data Engine Proxy Services Server is a new type of governance server -- not an OMAS.  So it's akin to Discovery Engine Services, Stewardship Services, etc which are all also dependencies

Reply:
ah! sure.